packages/keycodes/README.md

```diff
@@ -112,26 +138,25 @@ Keycode for LEFT key.
 Object that contains functions that return the available modifier
 depending on platform.
 
--   `primary`: takes a isApple function as a parameter.
--   `primaryShift`: takes a isApple function as a parameter.
--   `primaryAlt`: takes a isApple function as a parameter.
--   `secondary`: takes a isApple function as a parameter.
--   `access`: takes a isApple function as a parameter.
--   `ctrl`
--   `alt`
--   `ctrlShift`
--   `shift`
--   `shiftAlt`
+_Type_
+
+-   `null` 
```

Comment:
This is what I am referring to with the remarks in "Current Status" of the original comment.